vignettes/ORFikOverview.Rmd

```diff
@@ -23,8 +23,8 @@ ORFik currently supports:
 
 # Introduction
 
-`ORFik` main usage is to get GenomicRanges compatitable results of Open reading frame searching together with
-Ribo-seq and RNA-seq analysis. You can then get the features you want, and make various plots. 
+`ORFik` main usage is to get Open reading frames compatitable with Bioconductor's GenomicRanges package, combined with Ribo-seq and RNA-seq analysis.
```

Comment:
Open Reading Frames compatible with Bioconductor's GenomicRanges package, combined with Ribo-Seq, RNA-Seq and Cage-Seq data.